js/viz/core/utils.js

```diff
@@ -632,3 +632,18 @@ export function valueOf(value) {
 export function pointInCanvas(canvas, x, y) {
     return x >= canvas.left && x <= canvas.right && y >= canvas.top && y <= canvas.bottom;
 }
+
+export const getNextDefsSvgId = (function() {
+    let numDefsSvgElements = 1;
+    return function() { return 'DevExpress_' + numDefsSvgElements++; };
+})();
```

Comment:
I don't write this IIFE, I only move it from renderer. And I agree that strange code and long time ago (I think 6 years ago maybe or more) I wrote this function like you write it, but then it was writing like you see now. Why? I don't know. I did not touch it because there was a sense in such writing. Maybe @alexanderPolosatov can remember why it was write like this.